hub/codec/codecs.py

```diff
@@ -0,0 +1,380 @@
+from io import BytesIO
+import numcodecs
+from numcodecs.abc import Codec
+
+import numpy as np
+from PIL import Image
+
+
+class Base:
+    """Base class for numcodec compressors"""
+
+    def encode(self, array: np.ndarray) -> bytes:
+        raise NotImplementedError()
+
+    def decode(self, bytes: bytes) -> np.ndarray:
+        raise NotImplementedError()
+
+
+class NumPy(Base):
+    """Numpy compressor"""
+
+    def __init__(self):
+        super().__init__()
+
+    def encode(self, array: np.ndarray) -> bytes:
+        """
+        Encode given array
```

Comment:
Could you explain a bit more what this codec does? Is it using some intrinsic compression provided by numpy, or is it just a default "no-op" compressor?

Reply:
It mostly just pickles and unpickles arrays.